docs/copied-from-beats/shared-ssl-config.asciidoc

```diff
@@ -4,7 +4,9 @@
 You can specify SSL options when you configure:
 
 * <<configuring-output,outputs>> that support SSL
+ifeval::["{beatname_lc}"!="apm-server"]
```

Comment:
This has been added to my Beats PR: https://github.com/elastic/beats/pull/9595/commits/28f90229e60f028619b249c58700983574ba2f6d